packages/server/customer-os-api/entity/billed_type.go

```diff
@@ -0,0 +1,33 @@
+package entity
+
+type BilledType string
+
+const (
+	BilledTypeNone     BilledType = ""
+	BilledTypeMonthly  BilledType = "MONTHLY_BILLED"
```

Comment:
In DB I have these values:
"MONTHLY", "ANNUALLY", "ONCE"

Reply:
okay fixing :)